pkgs/development/tools/gomacro/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "gomacro";
+  version = "2.7";
+
+  src = fetchFromGitHub {
+    owner = "cosmos72";
+    repo = "gomacro";
```

Comment:
Common practice

```suggestion
    repo = pname;
```

Reply:
Actually I don't think this should be done everywhere. It makes it harder to open the github page when you try to find the download.